ci-operator/step-registry/cucushift/installer/rehearse/aws/cases/valid-endpoints/cucushift-installer-rehearse-aws-cases-valid-endpoints-workflow.yaml

```diff
@@ -1,7 +1,10 @@
 workflow:
   as: cucushift-installer-rehearse-aws-cases-valid-endpoints
-  steps: 
+  steps:
     pre:
-    - chain: cucushift-installer-rehearse-aws-cases-valid-endpoints-provision
+      - chain: cucushift-installer-rehearse-aws-cases-valid-endpoints-provision
+      - ref: cucushift-installer-reportportal-marker
+    post:
+      - ref: junit2reportportal
```

Comment:
The main purpose of this PR is

- Add step `cucushift-installer-reportportal-marker` as the last pre step.
- Add step `junit2reportportal` as the last post step.

IMO this is a special case. Not mean the change is correct or not, just mean it's a special case than the others as it originally does not have a post stage.  We may have other special cases, too.

Reply:
I think you misread my point.   I don't care which tool/cmd is used.  My point is that I'd like to review this PR, but currently changes made me impossible to do that.

Reply:
It is a batch update from a reliable yq tool, I just want to make my daily work easier. 

In this PR, you suggested only update `Add step cucushift-installer-reportportal-marker as the last pre step` and `Add step junit2reportportal as the last post step`, but not update yaml format to help your review. But it is not easy to achieve that among hundreds of yaml files. Especially how to ensure all the pre stage include `cucushift-installer-reportportal-marker` and how to ensure all the AWS workflow post stage include `junit2reportportal`.

Reply:
Thanks for your understanding.